README.md

```diff
@@ -27,4 +27,19 @@ to run `site-scrapers/MAImmunizations.js`
 1. When you're ready to deploy via AWS Lambda, run `npm run predeploy` which will generate `lambda.zip` for you. This needs to stay under 50 MB for you to upload it manually.
 1. Your production environment needs to have the environment variables `AWSS3BUCKETNAME`, `AWSACCESSKEYID`, and `AWSSECRETACCESSKEY` so that it can publish to S3. If you are inserting your own scrapers, set `PROPRIETARY_SITE_SCRAPERS_PATH` in production as well. If you have any scrapers that need to solve reCAPTCHAs, you will also need a `RECAPTCHATOKEN` from the 2captcha service.
 
+## Adding a scraper
+To scrape data from a site, you either need to:
+- figure out which API calls are made (Chrome devtools > Network tab) and make these calls yourself to fetch availability. This is preferable since the underlying APIs are less likely to change than the HTML on a given page.
+- interact with the site (clicking buttons, inspecting HTML elements, etc) using puppeteer
```

Comment:
Query: Is it indeed preferable to hunt for underlying APIs? One big problem is that is how we get in the situation where a site turns off access to users (e.g. unbookable appointments because of reservations) but we continue to report appointments that are actually unbookable.

I don't claim that Puppeteer is superior in all cases, but I am totally on the fence about which is better philosophically, and I hadn't realized we had made a project-wide decision on that.

Reply:
It's only my opinion haha, definitely wasn't a "project-wide decision". I can remove the line since it probably depends on the use case.